test/companies/companies_test.exs

```diff
@@ -15,6 +15,14 @@ defmodule Companies.CompaniesTest do
       assert %{page_number: 1, page_size: 16, total_entries: 2, total_pages: 1} = Companies.all()
     end
 
+    test "does not retrieve deleted records" do
+      insert(:company, is_deleted: true)
+      insert(:company, is_deleted: false)
```

Comment:
Since `false` is the default, do we need this _and_ line 21? I could see one or the other. If we want > 1 record we can use the `insert_pair` or `insert_list` function.

Reply:
Well, because I try to be paranoid in tests - it helps in weird cases like reversed logic.

Reply:
I'm not sure I follow. I'm just asking why this:

```elixir
insert(:company, is_deleted: false)
insert(:company)
```

Versus this, which is the same (right?):

```elixir
insert_list(2, :company, is_deleted: false)
```